### PR TITLE
Allowing access to location in Airplane mode with GPS on

### PR DESCRIPTION
**Pull request by manojdcoder**
#108: After lot of research at my end, I think this is the only way to handle GPS in airplane mode.

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "nativescript-geolocation",
-    "version": "4.2.3",
+    "version": "4.3.0",
```
Comment (DimitarTachev) [inline src/package.json]: As far as I see this is a bug fix, not a feature. It could be just 4.2.4.

**Comment by manojdcoder** [comment]
@DimitarTachev Thanks for taking time to review, have pushed the requested changes. Hope its good to merge now.

File: src/geolocation.android.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { Options, successCallbackType, errorCallbackType } from "./location-moni
 import * as permissions from "nativescript-permissions";
 
 declare var com: any;
+const LocationSettingsStatusCodes = com.google.android.gms.location.LocationSettingsStatusCodes;
```
[the remark this answers is not in the document]
Comment (manojdcoder) [reply src/geolocation.android.ts]: I have pushed the changes, thanks for letting me know about these issues. 

But just wondering still my app works fine with webpack and snapshot using same code. Is there a documentation that explains do / don'ts with webpack / snapshot?

**Comment by manojdcoder** [comment]
@DimitarTachev Can you please take a look and merge this, I could release use the updated version.

**Comment by manojdcoder** [comment]
Thank you @DimitarTachev Could you please push a build to NPM as well?

**Comment by DimitarTachev** [comment]
@manojdcoder We will try to publish it later today or tomorrow.

File: src/geolocation.android.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (DimitarTachev) [reply src/geolocation.android.ts]: It seems that we should improve the docs. Meanwhile, you could take a look at this blog post: https://www.nativescript.org/blog/improving-app-startup-time-on-android-with-webpack-v8-heap-snapshot

[the remark this answers is not in the document]
Comment (manojdcoder) [reply src/geolocation.android.ts]: Thank you @DimitarTachev but that blog also looks like very old, by at least it says Webpack + Snapshot is not possible / Snapshot not available for Angular etc., I hope that is not true now?

Hope to see some guidelines in appropriate docs soon - https://docs.nativescript.org/best-practices/bundling-with-webpack#v8-heap-snapshot

Thanks a lot for taking time to response.

**Pull request merged**